packages/app/src/app/pages/Sandbox/Editor/Workspace/elements.ts

```diff
@@ -122,6 +122,14 @@ export const Container = styled.div`
   width: 100%;
   overflow-y: overlay;
   overflow-x: auto;
+
+  ${props =>
+    props.REDESIGNED_SIDEBAR &&
+    css`
+      * {
+        box-sizing: border-box;
+      }
+    `}
```

Comment:
Do you need this for components/* or other elements too?

Reply:
For all, that should be in the root tbh